scripts/deploy.sh

```diff
@@ -0,0 +1,139 @@
+  kubectl apply -f ../kubernetes/keytransparency-deployment.yml
+}
+
+function initGcloud()
+{
+  gcloud --quiet version
+  gcloud auth activate-service-account --key-file ../service_key.json
+  # This might fail locally but is necessary on travis:
+  gcloud --quiet components update kubectl
+  gcloud config set project ${PROJECT_NAME}
+  gcloud config set compute/zone us-central1-a
+  gcloud container clusters get-credentials ci-cluster
+}
+
+function buildDockerImgs()
+{
+  # Build all images defined in the docker-compose.yml:
+  (cd ../ && docker-compose build)
+  # Separately build the DB:
+  # Work around some git permission issues on linux:
+  chmod a+r ../../trillian/storage/mysql/storage.sql
+  # Create and push a fresh db image (with schema created)
+  docker build -t  us.gcr.io/${PROJECT_NAME}/db -f ../kubernetes/mysql-trillian/Dockerfile ../..
+}
+
+function pushTrillianImgs()
+{
+  gcloud docker -- push us.gcr.io/${PROJECT_NAME}/db
+  images=("trillian_log_server" "trillian_map_server" "keytransparency-server" \
+  "trillian_log_signer" "keytransparency-signer")
+  for DOCKER_IMAGE_NAME in "${images[@]}"
+  do
+    docker tag ${DOCKER_IMAGE_NAME} us.gcr.io/${PROJECT_NAME}/${DOCKER_IMAGE_NAME}
+    # Push the images as we refer to them in the kubernetes config files:
+    gcloud docker -- push us.gcr.io/${PROJECT_NAME}/${DOCKER_IMAGE_NAME}
+  done
+}
+
+function pushKTImgs()
+{
+ gcloud docker -- push us.gcr.io/${PROJECT_NAME}/db
+  images=("keytransparency-server" "keytransparency-signer")
+  for DOCKER_IMAGE_NAME in "${images[@]}"
+  do
+    docker tag ${DOCKER_IMAGE_NAME} us.gcr.io/${PROJECT_NAME}/${DOCKER_IMAGE_NAME}
+    # Push the images as we refer to them in the kubernetes config files:
+    gcloud docker -- push us.gcr.io/${PROJECT_NAME}/${DOCKER_IMAGE_NAME}
+  done
+}
+
+function waitForTrillian()
+{
+  # Wait for trillian-map pod to be up:
+  COUNTER=0
+  MAPSRV=""
+  until [ -n "$MAPSRV" ] || [  $COUNTER -gt $MAX_RETRY ]; do
+    # Service wasn't up yet:
+    sleep 10;
+    let COUNTER+=1
+    MAPSRV=$(kubectl get pods --selector=run=trillian-map -o jsonpath={.items[*].metadata.name});
+  done
+
+  if [ -n "$MAPSRV" ]; then
+    echo "trillian-map service is up"
+  else
+    echo "Stopped waiting for trillian-map service. Quitting ..."
+    exit 1;
+  fi
+}
+
+function createTreeAndSetIDs()
+{
+  LOG_ID=""
+  MAP_ID=""
+  COUNTER=0
+  until [ -n "$LOG_ID" ] || [  $COUNTER -gt $MAX_RETRY ]; do
+    # RPC was not available yet, wait and retry:
+    sleep 10;
+    let COUNTER+=1
+    # get the currentl trillian-map pod:
+    MAPSRV=$(kubectl get pods --selector=run=trillian-map -o jsonpath={.items[*].metadata.name});
+    LOG_ID=$(echo 'go run $GOPATH/src/github.com/google/trillian/cmd/createtree/main.go --admin_server=localhost:8090 --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password="towel" --signature_algorithm=ECDSA --tree_type=LOG' | kubectl exec -i $MAPSRV -- /bin/sh )
```

Comment:
No, the command will be run directly on the pod (at the very end: ` kubectl exec -i $MAPSRV -- /bin/sh`
But we can change to `trillian-map` for consistency, if you prefer (which will resolve to the same machine as `localhost`).